certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6000 @@
+                      },
+                      "id": {
+                        "type": "string",
+                        "description": "Identifier of the domain.",
+                        "title": "Id"
+                      },
+                      "isAdminManaged": {
+                        "type": "boolean",
+                        "description": "Indicates if the domain is admin managed.",
+                        "title": "Admin managed",
+                        "enum": [
+                          true,
+                          false
+                        ]
+                      },
+                      "isDefault": {
+                        "type": "boolean",
+                        "description": "Indicates if this is the default domain.",
+                        "title": "Default",
+                        "enum": [
+                          true,
+                          false
+                        ]
+                      },
+                      "isInitial": {
+                        "type": "boolean",
+                        "description": "Indicates if this is the initial domain.",
+                        "title": "Initial",
+                        "enum": [
+                          true,
+                          false
+                        ]
+                      },
+                      "isRoot": {
+                        "type": "boolean",
+                        "description": "Indicates if this is the root domain.",
+                        "title": "Root",
+                        "enum": [
+                          true,
+                          false
+                        ]
+                      },
+                      "isVerified": {
+                        "type": "boolean",
+                        "description": "Indicates if the domain is verified.",
+                        "title": "Verified",
+                        "enum": [
+                          true,
+                          false
+                        ]
+                      },
+                      "supportedServices": {
+                        "type": "array",
+                        "items": {
+                          "type": "string"
+                        },
+                        "description": "Supported services."
+                      },
+                      "passwordValidityPeriodInDays": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Indicates the password validity in days.",
+                        "title": "Password validity"
+                      },
+                      "passwordNotificationWindowInDays": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Indicates the password notifcation window in days.",
+                        "title": "Password notification windows"
```

Comment:
Can you please correct spelling "notifcation"

Reply:
Fixed